gcp/table_gcp_compute_instance_group.go

```diff
@@ -259,14 +259,21 @@ func getComputeInstanceGroupInstances(ctx context.Context, d *plugin.QueryData,
 		return nil, err
 	}
 
-	resp, err := service.InstanceGroups.ListInstances(project, getLastPathElement(types.SafeString(instanceGroup.Zone)), instanceGroup.Name, &compute.InstanceGroupsListInstancesRequest{}).Do()
-
-	if err != nil {
-		plugin.Logger(ctx).Error("gcp_compute_instance_group.getComputeInstanceGroupInstances", "api_err", err)
-		return nil, err
+	if instanceGroup.Zone == "" {
```

Comment:
Hi @pdecat, could you please provide a comment explaining the code changes and why they are necessary? It would be helpful for future development.

Reply:
Sure, done!